source/objects/cableClass.m

```diff
@@ -0,0 +1,283 @@
+classdef cableClass<handle
+    
+    %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    % The ``cableClass`` creates a ``cable`` object saved to the MATLAB
+    % workspace. The ``cableClass`` includes properties and methods used
+    % to define cable connections relative to other bodies.
+    % It is suggested that the mooringClass be used for connections between
+    % the bodies and global reference frame
+    %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    properties (SetAccess = 'public', GetAccess = 'public')%input file
+        name                    = 'NOT DEFINED'
+        k_0                     = 0                                             % (`float`) Linear PTO stiffness coefficient. Default = `0`.
+        c_0                     = 0                                             % (`float`) Linear PTO damping coefficient. Default = `0`.                                             % (`float`) Rotational PTO cable-to-body coupling damping coefficient. Default = `0`.
+        K                       = 0
+        C                       = 0
+        L0                      = 0
```

Comment:
Remove equilibrium length, and have users define ``preTension`` as they would in the ``ptoClass`` or ``mooringClass``

Reply:
I strongly prefer having equilibrium length since this is the most physically meaningful quantity that one might have readily available to model a cable coupling two bodies, and is the critically important figure in determining whether or not the cable is slack. I am open to adding a preTension argument, but note that this will really complicate the mask functions for the dummy body positions. After the last group conversation I ended up chasing myself in circles thinking about which was the more intuitive option to include and kept landing on equilibrium length. Since this block is intended to couple two dynamic bodies, not body-to-seabed, equilibrium length is the parameter that requires the fewest number of assumptions.

Reply:
Yes good point, the set-up makes L0 an important parameter. I don't have an issue with using it, aside from trying to be consistent with how PTO and Mooring are initialized. Maybe keep the equilibrium length variable, but make it private and calculate based on the combination of constraint locations and the pretension. Then only let users define pretension for consistency with the other class parameters.

Reply:
I'm good with that, basically calculating equilibrium length from user-specified preTension and constraint locations, for consistency with other classes.